### DISCUSSION TITLE
Min-date is today for Date Field?

### DISCUSSION THREAD

**Discussion by hungcrush** [pocketbase/pocketbase#1977] [Q&A] [answered]
Seems it's missing select `today` for  min-date or max-date  feature. Do you have any idea for it?

![CleanShot 2023-03-05 at 18 46 37](https://user-images.githubusercontent.com/8098916/222958745-5b50cef9-4e15-4b24-a074-baee4a59c9f0.jpg)

**Comment by ganigeorgiev** [comment, marked as answer]
This is not supported via the field option constraints (and I'm not sure how this will work with the update action).

If you want to apply a dynamic "Min date" constraint on create, you can set a Create API rule like: `@request.data.expired > @now`.

**Reply by hungcrush** [reply]
Thank you. I got it
